contracts/zero-ex/CHANGELOG.json

```diff
@@ -1,4 +1,13 @@
 [
+    {
+        "version": "0.39.2",
+        "changes": [
+            {
+                "note": "Uprgade dependencies"
+            }
+        ],
+        "timestamp": 1680893079
```

Comment:
I think publish action will probably error out if timestamp is present. 

the same probably applies to other `CHANGELOG.json`s